app/utils/workflow.js

```diff
@@ -46,9 +46,9 @@ export default class WorkflowUtil {
     // we don't count the 3 periods towards the max length (meaning, our resulting label will
     // technically go over the max length we detect).
     // eslint-disable-next-line prettier/prettier
-    const beginningPart = name.substring(0, (Constants.MAX_GRAPH_LABEL_LENGTH / 2));
+    const beginningPart = name.substring(0, Constants.MAX_GRAPH_LABEL_LENGTH / 2);
```

Comment:
If this change came from prettier, I'm okay with the suggestion, but please remove the eslint override on the previous line.